src/components/button/button.style.js

```diff
@@ -62,7 +63,7 @@ function stylingForType({
     }
 
     & ~ & {
-      margin-left: 16px;
+      margin-left: ${fullWidth ? '0px;' : '16px;'};
```

Comment:
Is it possible to not set `margin-left` when `fullWidth` instead of `0px`?

Reply:
```
      ${fullWidth && css`margin-left: 16px`};
```

Reply:
Made the change as requested. I agree, it's a neater way of writing it. Thanks.